v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,1781 @@
+        },
+        factoryReset: { time: '1970-01-01T00:00:00Z', timestamp: 0 },
+        highActivity: false,
+        remoteControl: false,
+        identification: {
+            tag: { environmentId: 'cddd8855-ff50-4bbe-bb82-62b5057fa4f4' }, // this is the environment ID that you will receive from the SuperTokens team
+            url: 'http://example.com/index.html?eid=cddd8855-ff50-4bbe-bb82-62b5057fa4f4', // this is the URL that has been used to generate the request ID
+            linkedId: null,
+            timeInMS: 1723130887458,
+            incognito: false,
+            requestId: '1723130887451.92r32x', // this is the request ID that has been generated on the frontend
+            visitorId: 'mEYaqlY67Z55cHgzt37y',
+            confidence: { score: 1 },
+            browserDetails: {
+                os: 'Mac OS X',
+                device: 'Other',
+                osVersion: '10.15.7',
+                userAgent:
+                'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36',
+                browserName: 'Chrome',
+                browserFullVersion: '127.0.0',
+                browserMajorVersion: '127',
+            },
+        },
+        virtualMachine: false,
+        privacySettings: false,
+        locationSpoofing: false,
+        rawDeviceAttributes: {
+            // These are the raw device attributes that are being sent from the frontend
+            // They might vary based on the device and browser that is being used
+            audio: { value: 124.04346607114712 },
+            fonts: {
+                value: ['Arial Unicode MS', 'Gill Sans', 'Helvetica Neue', 'Menlo'],
+            },
+            canvas: {
+                value: {
+                Text: '32a115bd05e0f411c5ecd7e285fd36e2',
+                Winding: true,
+                Geometry: 'd45e7d71dc99e368affd8a40840c833d',
+                },
+            },
+            contrast: { value: 0 },
+            cpuClass: {},
+            colorDepth: { value: 124.04346607114712 },
+            colorGamut: { value: 'p3' },
+            architecture: { value: 127 },
+            cookiesEnabled: { value: true },
+        },
+    }
+};
+```
+</details>
+
+## Retrieving the Request ID
+
+We pass a `requestId` during the email password sign in, sign up and reset password APIs from the frontend (see [frontend setup section](frontend-setup#generating-a-request-id-on-the-frontend)). You can retrieve this ID by overriding these APIs in the emailpassword recipe on the backend.
+
+Below is an example of how to retrieve the request ID from the input body:
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```tsx
+    const signInPOST = async function (input: {
+        options: {
+            req: {
+                getJSONBody: () => Promise<{
+                    requestId: string;
+                }>;
+            };
+        };
```

Comment:
this can be very misleading, cause this isn't actual code that users should copy / paste. So you can either add the whole thing like emailpassword.init etc etc.. or remove this section since this is anyway there below.

Reply:
done